src/ProjectManagement/NuGetProject.cs

```diff
@@ -15,7 +15,7 @@ public abstract class NuGetProject
         protected NuGetProject() : this(new Dictionary<string, object>()) { }
         protected NuGetProject(IDictionary<string, object> metadata)
         {
-            if(metadata == null)
+            if (metadata == null)
             {
                 throw new ArgumentNullException("metadata");
```

Comment:
Did you intend to change this to nameof(metadata) as well? If not, we can always fix this using re-sharper